modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/routes/MDFRequestForm/steps/Review/components/Table/Table.tsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import ClayTable from '@clayui/table';
+
+interface ITable {
+	title?: string;
+	value?: string;
```

Comment:
```suggestion
	title: string;
	value: string;
```

Reply:
i cant do this change because sometimes the value is a object and your value is undefined

![image](https://user-images.githubusercontent.com/106164455/185399010-88cff3b8-e78e-4bb9-a966-34a0051dd2fc.png)